azure-functions/acbs-function/mappings/facility/facility-guarantee-amend.js

```diff
@@ -0,0 +1,26 @@
+const facilityGuaranteeAmend = (amendment) => {
```

Comment:
Could we add some documentation like the other functions, describing the related fields? e.g

```
/*
  "guaranteedLimit"      Description
  "expirationDate"       Description
*/
```

Reply:
Great shout -- added

Reply:
Nice one @AlexBramhill , not sure if we should follow the same pattern as other ACBS functions I can see where they have non-jsdoc style documentation at the top, seems the list the ACBS specific fields and what they are?

Reply:
I've added it as JSDocs. I think this is ultimately better and does not impact the code for the worst by introducing any patterns which will cause issue if not implimented everywhere... lmk what you think